ui/src/common/DateTimeSelector.tsx

```diff
@@ -13,6 +15,29 @@ interface DateTimeSelectorProps {
 
 export const DateTimeSelector: React.FC<DateTimeSelectorProps> = React.memo(
     ({selectedDate, onSelectDate, showDate, label, popoverOpen = () => {}}) => {
+        const {done, datetimeInputStyle} = useSettings();
+
+        if (!done) {
+            return <span>...</span>;
+        }
+
+        if (datetimeInputStyle === DatetimeInputStyle.Standard) {
+            const formatDate = (d: Date): string => {
+                return [d.getFullYear(), '-', d.getMonth(), '-', d.getDay(), 'T', d.getHours(), ':', d.getMinutes()]
```

Comment:
This doesn't work. getDay returns the day of the week, getMonth starts with 0. Use `selectedDate.format('YYYY-MM-DDTHH:mm')` instead.